src/CreateVFE.tsx

```diff
@@ -120,16 +169,109 @@ function CreateVFEForm({ onCreateVFE, header }: CreateVFEFormProps) {
             startAdornment: <AttachFileIcon />,
           }}
         />
-        <PhotosphereCenterFieldset
-          setPhotosphereCenter={setPhotosphereCenter}
+        <MuiFileInput
+          placeholder="Upload Navigation Map(optional)"
+          value={navMapFile}
+          onChange={(file) => {
+            if (file) {
+              setNavMapFile(file);
+              const url = URL.createObjectURL(file);
+              void handleNavMapChange(url);
+            }
+          }}
+          inputProps={{ accept: "image/*" }}
+          InputProps={{
+            startAdornment: <AttachFileIcon />,
+          }}
         />
         <Button
-          sx={{ margin: "auto" }}
-          variant="contained"
-          onClick={handleCreateVFE}
+          variant="outlined"
+          onClick={() => {
+            setMapDialogOpen(true);
+          }}
         >
-          Create
+          Select Photosphere Location
```

Comment:
Maybe this button should be hidden if there is no photosphere?

It's not very useful to open a dialog that only says "No map available".